ci/do_ci.sh

```diff
@@ -5,6 +5,7 @@ set -e
 export CC=gcc-4.9
 export CXX=g++-4.9
 export HEAPCHECK=normal
+export PPROF_PATH=/usr/bin/google-pprof
```

Comment:
IIRC pprof is already in the container because we build gperftools. I think it's in /thirdparty_build/bin/pprof. So I think path just needs to be set there (please confirm path). This is better than installing the package since it's guaranteed to match the version of gperftools we are using.